packages/react/src/elements/components/GlobalNav/TopNav/Notifications.js

```diff
@@ -119,9 +114,9 @@ Notifications.propTypes = {
    */
   title: PropTypes.string,
   /**
-   * An object containing props for a Notification, to be styled as a featured notification
+   * A Notification component, to be styled as a featured notification
    */
-  featuredNotification: PropTypes.shape(Notification.propTypes)
+  featuredNotification: PropTypes.element
```

Comment:
This is the minimum to make React happy. However, it's possible to go one step further and write a custom validator to ensure the element is specifically a Notification. Is it worth being so strict?

Reply:
A custom one would be helpful to developers. Give it a shot.